terraform/fdi_failure_sns_topic.tf

```diff
@@ -0,0 +1,47 @@
+# ------------------------------------------------------------------------------
+# Create the SNS topic that allows email to be sent when the findings-data-import
+# Lambda has a failure when processing an uploaded JSON.
+# ------------------------------------------------------------------------------
+
+resource "aws_sns_topic" "fdi_failure_alarm" {
+  name         = format("fdi-failure-alarms_%s", local.production_workspace ? "production" : terraform.workspace)
+  display_name = "findings-data-import_failure_alarm"
```

Comment:
Should we alphabetize these?

Reply:
If you are amenable I would like to punt that off into a separate PR to reformat _all_ of the Terraform as one cohesive piece of work.